sdk/attestation/azure-security-attestation/src/private/crypto/inc/crypto.hpp

```diff
@@ -135,6 +135,38 @@ namespace Azure { namespace Security { namespace Attestation { namespace _detail
       virtual std::string GetThumbprint() const = 0;
     };
 
+    /**
+     * @brief Convert a base64 encoded value to the PEM encoded equivalent.
+     *
+     * @param base64 base64 encoded value.
+     * @param pemType Type of the object to be converted - typically "CERTIFICATE", "PRIVATE KEY",
+     * or "PUBLIC KEY".
+     * @return std::string PEM encoded representation of the base64 value.
+     */
+    static std::string PemFromBase64(std::string const& base64, std::string const& pemType)
```

Comment:
nit.
I would really like to have this `impl` in a cpp, but I won't make too much noise about it.